src/queries/queries.js

```diff
@@ -15,21 +15,68 @@ import {
 //Probably not going to work, first attempt at queries
 export const GET_USER_INFO = (id) => gql`
 query GetUserInfo {
-  userId(id: ${id}) {
-    name
-    posts
-    fixedFollowers
-    fluxFollowers
+  user(id: ${id}) {
+    id
+    userName
+    firstName
+    lastName
+    email
+    phoneNumber
+    birthday
+    posts{
+      id
+      content
+      createdAt
+    }
+    followers{
+      friendId
+    }
+  }
+}
+`;
+
+export const GET_USER_NAME = (id) => gql`
+query GetUserInfo {
+  user(id: ${id}) {
+    userName
+    firstName
+    lastName
   }
 }
 `;
 
+// Get Followers
+export const GET_FOLLOWER_INFO = (id) => gql`
+query GetFollowerInfo {
+  usersFollowers(id: ${id}) {
```

Comment:
I wish there was a way we could refactor this in GQL to reduce these calls. They're nearly the same except for the first line after the query.